.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+*Please edit the list to your PR scope, then check off the boxes as you go!*
+
+## Contributor Checklist
+
+- [ ] I have broken down my PR scope into the following TODO tasks
+   -  [ ] task 1
+   -  [ ] task 2
+- [ ] I have passed with at least one python version with `circleci local execute`
+- [ ] I have verified all the required circleci jobs are passed without warnings
+- [ ] I have gotten :dancer: from my reviewer(s)
+
+## Post-merge Checklist
+- [ ] I have clicked the `Delete Branch` button after merge
```

Comment:
I'd take out 

I have verified all the required circleci jobs are passed without warnings (that's already a big red ❌  down below)

I have gotten :dancer: from my reviewer(s) (internal folks know that, external folks can't merge anyway)

I have clicked the `Delete Branch` button after merge (much as I want to encourage internal folks to do that, this is also meaningless to external folks on forks)